primeqa/pipelines/README.md

```diff
@@ -35,31 +40,37 @@ retriever.load()
 
 - Step 2:  Initialize the reader model. 
 
+Alternatively, we can use a reader based on ChatGPT.
+
 ```python
-reader = ExtractiveReader(model_name_or_path = model)
+reader = PromptGPTReader(api_key='API KEY HERE', model_name="gpt-3.5-turbo")
```

Comment:
Add link to API here?

Reply:
There is a link to the api key just above the code:
`We support integration with the OpenAI's ChatGPT(gpt-3.5-turbo) and InstructGPT(text-davinci-003), using an OpenAI API key (https://platform.openai.com/account/api-keys).`